Tasks/Maven/mavencodeanalysis.ts

```diff
@@ -142,16 +142,11 @@ function processAndAssignAnalysisResults(enabledCodeAnalysisTools:string[], modu
 }
 
 // Create a build summary from the analysis results of modules
-function createAndUploadBuildSummary(enabledTools:string[], modules:ma.ModuleAnalysis[]):void {
+function createAndUploadBuildSummary(enabledTools:string[], modules:ModuleAnalysis[]):void {
     var buildSummaryLines:string[] = [];
-    var totalViolationsInBuild = 0;
 
     enabledTools.forEach((toolName:string) => {
-        var toolAnalysisResults:ar.AnalysisResult[] = getToolAnalysisResults(modules, toolName);
-
-        toolAnalysisResults.forEach((toolAnalysisResult:ar.AnalysisResult) => {
-            totalViolationsInBuild += toolAnalysisResult.totalViolations;
-        });
```

Comment:
Non-structural refactor - moved the calculation of total violations out of this method to make it more readable.